<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAlbum</class>
 <widget class="QMainWindow" name="CAlbum">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>680</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblPhoto">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>lblPhoto</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="saPhotosMini">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1274</width>
         <height>100</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="lblPhotoMini_1">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lblPhotoMini_2">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="lblPhotoMini_3">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_3</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="lblPhotoMini_4">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_4</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="lblPhotoMini_5">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_5</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="lblPhotoMini_6">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_6</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QLabel" name="lblPhotoMini_7">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_7</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QLabel" name="lblPhotoMini_8">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_8</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="8">
           <widget class="QLabel" name="lblPhotoMini_9">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_9</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QLabel" name="lblPhotoMini_10">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>80</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="text">
             <string>lblPhotoMini_10</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actFile_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actEdit_First"/>
    <addaction name="actEdit_Prior"/>
    <addaction name="actEdit_Next"/>
    <addaction name="actEdit_Last"/>
    <addaction name="actEdit_GoTo"/>
    <addaction name="separator"/>
    <addaction name="actEdit_SaveAs"/>
    <addaction name="actEdit_Insert"/>
    <addaction name="actEdit_Delete"/>
    <addaction name="actEdit_Edit"/>
    <addaction name="actEdit_SetPrimary"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFile_Exit"/>
   <addaction name="separator"/>
   <addaction name="actEdit_First"/>
   <addaction name="actEdit_Prior"/>
   <addaction name="actEdit_Next"/>
   <addaction name="actEdit_Last"/>
   <addaction name="actEdit_GoTo"/>
   <addaction name="separator"/>
   <addaction name="actEdit_SaveAs"/>
   <addaction name="actEdit_Insert"/>
   <addaction name="actEdit_Delete"/>
   <addaction name="actEdit_Edit"/>
   <addaction name="actEdit_SetPrimary"/>
  </widget>
  <action name="actFile_Exit">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Exit.bmp</normaloff>:/Menu/Menu/Exit.bmp</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actEdit_First">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/First.bmp</normaloff>:/Menu/Menu/First.bmp</iconset>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </action>
  <action name="actEdit_Prior">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Prior.bmp</normaloff>:/Menu/Menu/Prior.bmp</iconset>
   </property>
   <property name="text">
    <string>Prior</string>
   </property>
  </action>
  <action name="actEdit_Next">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Next.bmp</normaloff>:/Menu/Menu/Next.bmp</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actEdit_Last">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Last.bmp</normaloff>:/Menu/Menu/Last.bmp</iconset>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </action>
  <action name="actEdit_GoTo">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/GoTo.bmp</normaloff>:/Menu/Menu/GoTo.bmp</iconset>
   </property>
   <property name="text">
    <string>Go to...</string>
   </property>
   <property name="toolTip">
    <string>Go to...</string>
   </property>
  </action>
  <action name="actEdit_Insert">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Insert.bmp</normaloff>:/Menu/Menu/Insert.bmp</iconset>
   </property>
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actEdit_Delete">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Delete.bmp</normaloff>:/Menu/Menu/Delete.bmp</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actEdit_Edit">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/Edit.bmp</normaloff>:/Menu/Menu/Edit.bmp</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actEdit_SaveAs">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/SaveAs.bmp</normaloff>:/Menu/Menu/SaveAs.bmp</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save As</string>
   </property>
  </action>
  <action name="actEdit_SetPrimary">
   <property name="icon">
    <iconset resource="../Resources/App.qrc">
     <normaloff>:/Menu/Menu/SetPrimaryImage.bmp</normaloff>:/Menu/Menu/SetPrimaryImage.bmp</iconset>
   </property>
   <property name="text">
    <string>Set primary</string>
   </property>
   <property name="toolTip">
    <string>Set primary</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resources/App.qrc"/>
 </resources>
 <connections/>
</ui>
